### ISSUE TITLE
`aria-expanded` must be explicit (`true`/`false`)

### ISSUE THREAD

**Issue by thierryk** [tabpanelwidget/tabpanelwidget#14]
Thank you for reporting this.

The state of the  `aria-expanded` attribute should be **explicit** (`true` / `false`).
Per spec, when it is **undefined (default)**:

> The element, or another grouping element it controls, is neither expandable nor collapsible; all its child elements are shown or there are no child elements.

We'll update the script.

_Originally posted by @thierryk in https://github.com/tabpanelwidget/tabpanelwidget/discussions/13#discussioncomment-2335813_

**Comment by thierryk** [comment]
When the widget is **displayed as an Accordion**, we should have the following when the panel associated to the `role="button"` is expanded:

```
<span ... role="button" aria-expanded="true"> ... </span>
```

And this when the associated panel is closed:

```
<span ... role="button" aria-expanded="false"> ... </span>
```

Right now, the `aria-expanded` attribute is **not added** to the node if its associated panel is closed.
